features/spicy-protocol-analyzer/testing/tests/standalone.spicy

```diff
@@ -0,0 +1,7 @@
+# @TEST-EXEC: spicyc ${DIST}/analyzer/@ANALYZER_LOWER@.spicy -j -d -o @ANALYZER_LOWER@.hlto
+# @TEST-EXEC: printf "test string" | spicy-dump -p @ANALYZER@::@UNIT_ORIG@ @ANALYZER_LOWER@.hlto >output 2>&1
+# @TEST-EXEC: TEST_DIFF_CANONIFIER= btest-diff output
+#
+# @TEST-DOC: Test parsing behavior of @ANALYZER@.
+
+# TODO: Add standalone parsing tests here.
```

Comment:
Not added here, but maybe let's clean up the order of docs and comments here to make clearer where users should edit, e.g.,

```suggestion
# @TEST-DOC: Test parsing behavior of @ANALYZER@.

# @TEST-EXEC: spicyc ${DIST}/analyzer/@ANALYZER_LOWER@.spicy -j -d -o @ANALYZER_LOWER@.hlto

# TODO: Add standalone parsing tests below.
# @TEST-EXEC: printf "test string" | spicy-dump -p @ANALYZER@::@UNIT_ORIG@ @ANALYZER_LOWER@.hlto >output 2>&1
# @TEST-EXEC: TEST_DIFF_CANONIFIER= btest-diff output
```